test/jasmine/tests/lib_test.js

```diff
@@ -1522,7 +1522,12 @@ describe('Test lib.js:', function() {
                 ['-100.001', -100.001],
                 ['  $4.325  #%\t', 4.325],
                 [' " #1" ', 1],
-                [' \'\n \r -9.2e7   \t\' ', -9.2e7]
+                [' \'\n \r -9.2e7   \t\' ', -9.2e7],
+                ['1,690,000', 1690000],
+                ['1 690 000', 1690000],
+                ['2 2', 22],
```

Comment:
Oh good call - `'1 690 000'` will show up. `'2 2'` looks weird, and likely to be a parsing error, but I don't see how we can distinguish it from legitimate spaces in numbers.